Objective-C/CBLLog.h

```diff
@@ -35,7 +35,7 @@ NS_ASSUME_NONNULL_BEGIN
 /** File logger writing log messages to files. */
 @property (readonly, nonatomic) CBLFileLogger* file;
 
-/** For setting a custom logger. */
+/** For setting a custom logger. Set log level for custom instance, before setting this property. */
 @property (nonatomic, nullable) id<CBLLogger> custom;
```

Comment:
How about something like this?

```
Changing the log level of the assigned custom logger will require the custom logger to be reassigned so that the change can be affected.
```